controllers/secretagentconfiguration_controller.go

```diff
@@ -67,6 +64,9 @@ func (reconciler *SecretAgentConfigurationReconciler) Reconcile(req ctrl.Request
 	)
 
 	var instance v1alpha1.SecretAgentConfiguration
+	var sm *secretsmanager.SecretManager
```

Comment:
Since this is an interface, we don't need a pointer here.

Reply:
Doesn't seem to work without it?

Reply:
Just pushed a commit removing the pointer.  Look better?